cmd/manager/main.go

```diff
@@ -182,8 +192,10 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&flags.config, "meshConfig", cmd.DefaultConfigMapName,
 		fmt.Sprintf("ConfigMap name for Istio mesh configuration, key should be %q", cmd.ConfigMapKey))
 
-	discoveryCmd.PersistentFlags().IntVarP(&flags.sdsPort, "port", "p", 8080,
+	discoveryCmd.PersistentFlags().IntVarP(&flags.sdsPort, "sdsPort", "p", 8080,
```

Comment:
This should probably be `discoveryPort` since it serves CDS/RDS as well. Also, this may cause integration test failures unless everything has been switched over to use the ConfigMap - @kyessenov, this should be a safe change from that perspective now, correct?